### PR TITLE
Add content for VS Code and .NET CLI to tutorial

### LINKED ISSUE #976: [New article]: CLI version of adding an existing project to Aspire
### Proposed topic or title

Add .NET Aspire to an existing .NET app (CLI)

### Location in table of contents.

https://learn.microsoft.com/en-us/dotnet/aspire/get-started/add-aspire-existing-app

### Reason for the article

We had feedback that without VS, there is no good explanation of what the VS stuff does and having a CLI pivot for this article would be helpful showing the specific steps.

### Article abstract

Using the dotnet CLI you can enable an existing app to .NET Aspire using a few CLI commands and changing some code.

### Relevant searches

_No response_

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Add content for VS Code and .NET CLI to tutorial.

Fixes #976

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/get-started/add-aspire-existing-app.md](https://github.com/dotnet/docs-aspire/blob/f380fbc6342e3dbbf5521cdff81789e169b65e52/docs/get-started/add-aspire-existing-app.md) | [Tutorial: Add .NET Aspire to an existing .NET app](https://review.learn.microsoft.com/en-us/dotnet/aspire/get-started/add-aspire-existing-app?branch=pr-en-us-1006) |

### CODE REVIEW

File: docs/get-started/add-aspire-existing-app.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-ms.date: 05/17/2024
+ms.date: 05/28/2024
 ms.topic: how-to
+zone_pivot_groups: dev-environment
 ---
 
 # Tutorial: Add .NET Aspire to an existing .NET app
 
-If you have already created a microservices .NET web app, you can use Visual Studio to add .NET Aspire to it and get all the features and benefits available to those who enabled .NET Aspire when they selected **File** > **New** > **Project** and chose **Enlist in Aspire orchestration**. In this article, you'll add .NET Aspire orchestration to a simple, pre-existing .NET 8 project. You'll learn how to:
+If you have already created a microservices .NET web app, you can add .NET Aspire to it and get all the features and benefits available to those who enabled .NET Aspire. In this article, you'll add .NET Aspire orchestration to a simple, pre-existing .NET 8 project. You'll learn how to:
```
Comment (CamSoper) [inline docs/get-started/add-aspire-existing-app.md:11]:
```suggestion
If you have already created a microservices .NET web app, you can add .NET Aspire to it and get all the features and benefits of .NET Aspire. In this article, you'll add .NET Aspire orchestration to a simple, pre-existing .NET 8 project. You'll learn how to:
```

File: docs/get-started/add-aspire-existing-app.md [hunk 2]
```diff
@@ -53,16 +56,127 @@ Open and start debugging the project to examine its default behavior:
...
+    ```
+
+    ## [Windows](#tab/windows)
+
+    ```json
+    {
+        "version": "0.2.0",
+        "compounds": [
+            {
+                "name": "Rull all",
```
Comment (CamSoper) [inline docs/get-started/add-aspire-existing-app.md:108]: wut?

**Review by CamSoper — CHANGES_REQUESTED** [review]

File: docs/get-started/add-aspire-existing-app.md (hunk 2 above)
Comment (IEvangelist) [reply docs/get-started/add-aspire-existing-app.md:108]:
```suggestion
                "name": "Run all",
```

File: docs/get-started/add-aspire-existing-app.md [hunk 3]
```diff
@@ -53,16 +56,127 @@ Open and start debugging the project to examine its default behavior:
...
+1. Copy and paste the following JSON into this file and **Save**:
+
+    ## [Unix](#tab/unix)
+
+    ```json
+    {
+        "version": "0.2.0",
+        "compounds": [
+            {
+                "name": "Rull all",
```
Comment (IEvangelist) [inline docs/get-started/add-aspire-existing-app.md:77]:
```suggestion
                "name": "Run all",
```

**Review by CamSoper — APPROVED** [review]

**Pull request merged**